7_3.py

```diff
@@ -0,0 +1,17 @@
+
+class Message:
+    """
+    A Message Representation Class
+
+    """
+    def __init__(self, title, body, sender, receiver):
+        self.title = title
+        self.body = body
+        self.sender = sender
+        self.receiver = receiver
+
+    def __str__(self):
+        return 'Message (title=' + self.title + ' ,body=' + self.body + ' ,sender=' + self.sender+ ' ,receiver=' + self.receiver+ ')'

```

Comment:
There is a more elegant way of inserting values into strings. I recommend using curly braces.
https://matthew-brett.github.io/teaching/string_formatting.html#:~:text=The%20curly%20braces%20show%20where,%3E%3E%3E